tests/test_logger.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+import time
+
+import six
+import unittest2 as unittest
+
+from mock import Mock
+from six.moves import StringIO
+
+from kobo.worker.logger import LoggingThread, LoggingIO
+from .utils import ArgumentIsInstanceOf
+
+
+class TestLoggingThread(unittest.TestCase):
+
+    @unittest.skipIf(six.PY3, 'Encoding errors on Python 3')
```

Comment:
Would you be able to file a bug about that?

Reply:
Yes, but on GitHub we already have a bug report about py3.

Reply:
Then you can mention it in the text here.

Reply:
Created issue #66 for this case.